site/docs/.vuepress/config.ts

```diff
@@ -323,6 +323,10 @@ export default defineUserConfig<DefaultThemeOptions>({
                     text: "Google Cloud Functions",
                     link: "/hosting/gcf.md",
                   },
+                  {
+                    text: "VPS",
```

Comment:
Should we give this a name that people understand who are not familiar with the abbreviation VPS?